### ISSUE TITLE
Update documentation for BAGIS V3

### ISSUE THREAD

**Issue by lbross** [PSU-CSAR/vb-bagis#35]
https://sites.google.com/site/nwccspatialservices/users-references/users-manuals
Add link for BAGIS V3 online manual
- Default settings button (web services)
- Auto site scenario tool
- Elev-precip tool
Modify BAGIS Tools online manual to include:
- eBAGIS menu; Make it a separate link

**Comment by lbross** [comment]
Users manual for BAGIS V3: https://sites.google.com/site/nwccspatialservices/users-references/users-manuals/bagis-v3-user-s-manual

**Comment by jdduh** [comment]
Looks good. I made some minor edits.

**Comment by lbross** [comment]
User manual for BAGIS Tools - eBAGIS menu:
https://sites.google.com/site/nwccspatialservices/users-references/users-manuals/bagis-tools/ebagis-aoi-menu

I think this completes the documentation tasks for BAGIS v3 but please let me know if I've missed anything.

**Comment by jdduh** [comment]
Do users need to create an account on the server first? We also need to provide instructions on how to get 
a permission to delete AOIs from the server. Are these related to the web site that Aida is creating? If so, we can wait until there is a beta release of eBAGIS.

**Comment by lbross** [comment]
@jkeifer Can you summarize the rules that govern delete permissions for AOIs? Can anyone delete their own AOI but only administrators delete someone else's AOI?

**Comment by lbross** [comment]
Users do need to create an account on the server first. Yes, we should provide documentation on how to create an account but this is dependent on awebsite. @jkeifer was working on the security web application but I'm not sure if it has been reassigned. We will leave this issue open until we have a beta release of the security application for screenshots.

**Comment by lbross** [comment]
Closing. The documentation is complete!
